vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -79,19 +79,26 @@ function Retry-Command {
 function Expand-OS-Partition {
     $customizedDiskSize = $env:CustomizedDiskSize
     if ([string]::IsNullOrEmpty($customizedDiskSize)) {
-        Write-Log "No need to expand the OS partition size, default size 30GB"
+        Write-Log "No need to expand the OS partition size, default size 30GB by parker"
```

Comment:
I think that it is not "by parker" (packer?). I think that the default os disk size depends on the image

Reply:
@ShiqianTao Where do you find the default size?

Reply:
@AbelHu @wanqingfu  Yes, the default value = 30 because the image suffix is "smalldisk": https://azure.microsoft.com/en-us/blog/new-smaller-windows-server-iaas-image/

Reply:
@ShiqianTao @AbelHu  When the custom size is smaller than 30 GB, "building windows vhd" fails, as seen in the log [here](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=55677908&view=logs&j=ceeb8f07-abdb-53f4-60d8-80121461daa9&t=9925fa4f-3645-534b-dffc-a869b4c81936). (OS_DISK_SIZE_GB = 25) When it is exactly 30 GB, the pipeline succeeded, shown [here](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=55677908&view=logs&j=da83e503-f0a7-5e39-4c4e-547ddc065d07&t=c25eccdb-c467-51a1-7682-f93363e2409c).